src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -408,6 +408,8 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     dustAmountCryptoBaseUnit: poolAssetFeeAssetDustAmountCryptoBaseUnit,
     outboundFeeCryptoBaseUnit,
   } = useSendThorTx({
+    // Asym asset withdraws are the only ones occurring an asset Tx - both sym and asym RUNE side withdraws occur a RUNE Tx instead
+    enableEstimateFees: Boolean(opportunityType === AsymSide.Asset),
```

Comment:
:+1:

Reply:
can do the same on the rune `useSendThorTx` for `opportunityType !== AsymSide.Asset` while we are at it

Reply:
334a8f5e77